django/template/loaders/eggs.py

```
@@ -20,19 +31,42 @@ def __init__(self, engine):
             raise RuntimeError("Setuptools must be installed to use the egg loader")
         super(Loader, self).__init__(engine)
 
+    def get_contents(self, origin):
+        try:
+            source = resource_string(origin.app_name, origin.pkg_name)
+        except:
```

bare except is okay here?